setup/nextcloud.sh

```diff
@@ -173,6 +177,14 @@ if [ ! -d /usr/local/lib/owncloud/ ] || [[ ! ${CURRENT_NEXTCLOUD_VER} =~ ^$nextc
 	if [ ! -z ${CURRENT_NEXTCLOUD_VER} ]; then
 		# Database migrations from ownCloud are no longer possible because ownCloud cannot be run under
 		# PHP 7.
+
+		if [ -e $STORAGE_ROOT/owncloud/config.php ]; then
+			echo "Remove read-onlyness temporarily for migrations ..."
```

Comment:
End users likely won't know what this means and will likely just cause more questions on the forums/github

Reply:
True. Removed.